torch/csrc/api/src/nn/modules/embedding.cpp

```diff
@@ -11,27 +11,72 @@
 namespace torch {
 namespace nn {
 
-EmbeddingOptions::EmbeddingOptions(int64_t count, int64_t dimension)
-    : count_(count), dimension_(dimension) {}
+EmbeddingOptions::EmbeddingOptions(int64_t num_embeddings, int64_t embedding_dim) : num_embeddings(num_embeddings), embedding_dim(embedding_dim) {}
 
-EmbeddingImpl::EmbeddingImpl(EmbeddingOptions options) : options(options) {
-  reset();
+EmbeddingImpl::EmbeddingImpl(EmbeddingOptions options): options(options) {
+  EmbeddingImpl::reset();
```

Comment:
We don't need to use `EmbeddingImpl::` here as `reset()` already calls the right function